products/BellHybrid/CMakeLists.txt

```diff
@@ -77,20 +77,37 @@ strip_executable(BellHybrid)
 
 include(AddBootBin)
 add_boot_bin(BellHybrid)
+
 add_image(
     PRODUCT BellHybrid
     SYSROOT sysroot
-    ASSETS  assets
+    LUTS Luts.bin
+    DEPENDS assets updater.bin-target ecoboot.bin-target BellHybrid-boot.bin BellHybrid-version.json-target
 )
 
 include(AddHexTarget)
 add_hex_target(BellHybrid)
 
 include(BinaryAssetsVersions.cmake)
 include(DownloadAsset)
-download_asset_json(${CMAKE_CURRENT_SOURCE_DIR}/assets.json ${PROJECT_SOURCE_DIR} ${MUDITA_CACHE_DIR})
-download_asset_release(PureUpdater_RT.bin updater.bin PureUpdater ${UPDATER_BIN_VERSION})
-download_asset_release(ecoboot.bin ecoboot.bin ecoboot ${ECOBOOT_BIN_VERSION})
+# copy all assets required to build catalog under current folder as in json recipe
+download_asset_json( json-target
+                     ${CMAKE_CURRENT_SOURCE_DIR}/assets.json
+                     # ${CMAKE_SOURCE_DIR}
+                     ${CMAKE_BINARY_DIR}/sysroot/sys/current/
+                     MuditaOSAssets
+                     ${MUDITA_CACHE_DIR}
+    )
+# copy all assets required to build catalog under current folder as in json recipe
+download_asset_release_json(    json-images-target
```

Comment:
Can it be possibly merged with the `download_asset_json`?

Reply:
it could be possibly done, but would require yet some more changes in the `json` description - as right now we pass repository as an argument - and we would need it in the `json` file. Not too hard to do - not required for these changes.
Here we have 2 repositories:
1. public one, with images etc
2. non public one: with luts.bin and fonts.
and it represents this split